bundles/mapping/mapmodule/domain/AbstractLayer.js

```diff
@@ -1262,19 +1264,31 @@ Oskari.clazz.define(
         getGeometryType: function () {
             return null;
         },
-        getVisibilityInfo () {
+        getVisibilityInfo: function () {
             return this._visibilityInfo;
         },
-        setVisibilityInfo (info) {
+        setVisibilityInfo: function (info) {
             this._visibilityInfo = info;
         },
-        updateVisibilityInfo (updated) {
+        updateVisibilityInfo: function (updated) {
             this._visibilityInfo = { ...this._visibilityInfo, ...updated };
         },
-        isVisibleOnMap () {
+        isVisibleOnMap: function () {
             const { unsupported, ...booleans } = this.getVisibilityInfo();
             return this.getOpacity() !== 0 && !unsupported &&
                 Object.values(booleans).every(b => b === true);
+        },
+        getDescribeLayerStatus: function () {
+            return this._describeLayerStatus;
+        },
+        setDescribeLayerStatus: function (status) {
+            this._describeLayerStatus = status;
+        },
+        requiresDescripeLayer: function () {
+            return false;
+        },
+        handleDescribeLayer: function (info) {
```

Comment:
I guess this could replace the modelbuilders that we have for layers currently 🤔 Need to think if we should just get rid of one or the other.

Reply:
Didn't have good feelings how I solved describe layer info handling. Had to add handling in too many places. Layer, builder and plugins may all require info. One solution is to remove builders since them doesn't have much work to do nowadays.